Various cosmetic changes and formatting

classicrocker883 opened the pull request:
<!--

Submitting a Pull Request

- Please fill out all sections of this form. You can delete the helpful comments.
- Pull Requests without clear information will take longer and may even be rejected.
- We get a high volume of submissions so please be patient during review.

-->

### Description
>[!NOTE]
> I suppose if needed to, these can be regrouped into another **PR**
- Most of files now used is `ANY`/`ALL` instead of `&&`/`||`
- Update spacing, syntax, comments, capitalize, spacing, and such to match likeness in code
- - This helps simplify code and adds better readability

<p>

For python scripts - mostly cosmetic/format changes, basically just hit `Format Document` and some tweaks:
- Reformatted many Python (.py) script files (for spacing and readability):
- - **buildroot\share\scripts**
- - **buildroot\share\PlatformIO\scripts**
- - **src\lcd\extui\ftdi_eve_touch_ui\ftdi_eve_lib\scripts**
Also updated comments, such as "help" and `usage:`, when running the script in a terminal
<p>

Some rearranging:
- Rename `fall-through`/`fall through` to correct `Fallthrough`
- Fix runout/filament sensor spacing
- Rearrange: **marlinui.h**, **planner.cpp**, **creality/dwin.cpp**
- Arrange `print_heater_state/s()` in **temperature.cpp** to be inline with Temperature sensor IDs (**src\inc\Conditionals-4-adv.h**)

<p>

Added:
- Group `*** BARICUDA ***` - **gcode.h**
- Use `GET_TEXT_F` and `MSG_...` from **language_en.h** instead of literal Strings in CrealityUI **dwin.cpp**, which was a `TODO:`

<p>

Removed:
- Remove redundant code:
- - Remove double nested `#if ENABLED(AUTO_BED_LEVELING_UBL)` from **settings.cpp**

<p>

#### Move from other **PR**
Updated **fwretract.h**|**fwretract.cpp**
- Arrange/improve values + comments
- Rearrange `fwretract_settings_t` struct to follow gcode example

<!--

Clearly describe the submitted changes with lots of details. Include images where helpful. Initial reviewers may not be familiar with the subject, so be as thorough as possible. You can use MarkDown syntax to improve readability with bullet lists, code blocks, and so on. PREVIEW and fix up formatting before submitting.

-->

### Requirements

<!-- Does this PR require a specific board, LCD, etc.? -->

### Benefits

<!-- What does this PR fix or improve? -->

### Configurations

<!-- Attach Configurations ZIP and any other files needed to test this PR. -->

### Related Issues

<!-- Does this PR fix a bug or fulfill a Feature Request? Link related Issues here. -->

classicrocker883 committed:
cosmetic changes

classicrocker883 committed:
utility.h IAXIS_CHAR

thinkyhead commented:
> Arrange `print_heater_state/s()` to be inline with Temperature sensor IDs

The multi-hotend states are sorted to the end for the convenience of hosts to get the current hotend and bed temperature in the early part of the string, so those should not be moved to the front or rearranged based on the arbitrary order of heaters in the configuration files or elsewhere.

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' into bugfix-2.1.x-October7

classicrocker883 committed:
revert print_heater_states

classicrocker883 committed:
update planner (forgot from last PR)

classicrocker883 commented:
@thinkyhead ok I reverted that part in **temperature.cpp**

I added a couple changes in the **planner** files which were missed in the recent merge commit.

the only thing i'm not so sure about it **utility.h** - `IAXIS_CHAR`, is that fine or should be reverted?

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' into bugfix-2.1.x-October7

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
Merge https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
cosmetic changes

classicrocker883 committed:
revert for fix

classicrocker883 committed:
update motion.h

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' into bugfix-2.1.x-October7

classicrocker883 committed:
fix extra init_lcd(), remove HAS_UI_UPDATE (redundant)

classicrocker883 committed:
arrrange quick_feedback()

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' into bugfix-2.1.x-October7

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' into bugfix-2.1.x-October7

classicrocker883 committed:
More formatting, revert some - move to other PR

classicrocker883 committed:
moved to other PR

classicrocker883 committed:
add fwretract (from other PR)

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
remove settings.cpp UBL redundancy (from other PR)

classicrocker883 committed:
add dwin.cpp formatting

classicrocker883 committed:
Merge https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
Fix, update Python scripts

classicrocker883 committed:
put back accidental deletion

classicrocker883 committed:
capitalize

classicrocker883 committed:
Merge https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-March6

classicrocker883 committed:
Merge https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
more formatting, reverted back a couple things

classicrocker883 committed:
more formatting

classicrocker883 committed:
update motion.cpp

classicrocker883 committed:
shorten some lines

classicrocker883 committed:
shorten

classicrocker883 committed:
more changes
revert some
format upload.py

classicrocker883 committed:
more changes

classicrocker883 committed:
rearrange + spacing - add remove between lines

classicrocker883 committed:
remove spacing

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
update scripts format

classicrocker883 committed:
update some spacing

classicrocker883 committed:
update spacing

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
update #define spaces
remove spaces in #define

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
Update and arrange

classicrocker883 committed:
revert some, add spacing to beginning

classicrocker883 committed:
fix creality/dwin.cpp

classicrocker883 committed:
arrange order M207-208 fwretract.cpp

classicrocker883 committed:
update some script spacing, commenting

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' into bugfix-2.1.x-October7

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' into bugfix-2.1.x-October7

classicrocker883 committed:
arrange fwretract

classicrocker883 committed:
fix error

classicrocker883 committed:
update comments

classicrocker883 committed:
update comment pinsDebug

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
some changes

classicrocker883 committed:
fix error: temp

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' into bugfix-2.1.x-October7

classicrocker883 committed:
update formatting - spacing/comments

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' into bugfix-2.1.x-October7

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
prefer Fallthrough

classicrocker883 committed:
prefer ALL/ANY

classicrocker883 committed:
fix temp.h

classicrocker883 committed:
more formatting from other PR

classicrocker883 committed:
undo mistake

classicrocker883 committed:
fix py spacing

classicrocker883 committed:
fix dogm syntax error

classicrocker883 committed:
revert `&& ENABLED()`

classicrocker883 committed:
revert `|| ENABLED()`

classicrocker883 committed:
update types.h

classicrocker883 committed:
specify `!defined()`

classicrocker883 committed:
use `seenval`

classicrocker883 committed:
adjust workflows spacing

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
Revert "prefer ALL/ANY", moved to another PR

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
fix `set_pwm_duty`

classicrocker883 committed:
swap/semi revert

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' into bugfix-2.1.x-October7

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
fix motion error

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' into bugfix-2.1.x-October7

classicrocker883 committed:
Update config.py

classicrocker883 committed:
Update config.py

classicrocker883 committed:
Merge branch 'bugfix-2.1.x' of https://github.com/MarlinFirmware/Marlin into bugfix-2.1.x-October7
